ocaml/idl/datamodel.ml

```diff
@@ -3442,6 +3442,10 @@ let vdi_resize_online = call
     ~name:"resize_online"
     ~in_oss_since:None
     ~in_product_since:rel_rio
+    ~lifecycle: [
+      Published, rel_rio, "Online resize vdi";
+      Removed, rel_inverness, "Remove feature: online resize vdi"
```

Comment:
Here we could explain that we removed this call because it was never supported. We could say something like:
```
Online VDI resize is not supported by any of the storage backends.
```